pycm/pycm_util.py

```diff
@@ -781,3 +781,31 @@ def inner_function(*args, **kwargs):
             raise pycmMatrixError(METRICS_OFF_ERROR)
         return func(*args, **kwargs)
     return inner_function
+
+
+def deprecated(func):
+    """
+    Send a warning regarding decorated function's deprication.
+
+    :param func: input function
+    :type func: function
+    :return: inner function
+    """
+    @wraps(func)
```

Comment:
First should import `wraps` from `functools` module

Reply:
It has been imported at the beginning of the file (line 11). Am I right?

Reply:
Yeah it's okay